pyproject.toml

```diff
@@ -55,3 +56,11 @@ CI-CD = [
   "cubitpy@git+https://github.com/imcs-compsim/cubitpy.git@main",
   "setuptools" # Needed for coverage-badge
 ]
+
+[tool.pytest.ini_options]
+addopts = "-p pytest_cov --cov-report=term --cov-report=html --cov-fail-under=30 --cov=meshpy/ --cov=tutorial/ --cov-append"
+markers = [
+  "fourc_arborx: tests in combination with 4C and ArborX",
```

Comment:
Later on this is named `4C_ArborX` -> does this have to be the same string?

Reply:
i just differentiated the case style because I liked lowercase better within the decorators. And I thought within the CLI this approach looks better 😅 They do not have to be identical

Reply:
That makes sense. In the CLI I also prefer `4C`.